impeller/aiks/picture.cc

```diff
@@ -52,7 +53,10 @@ std::shared_ptr<Texture> Picture::RenderToTexture(
 
   // This texture isn't host visible, but we might want to add host visible
   // features to Image someday.
-  auto target = RenderTarget::CreateOffscreen(*context.GetContext(), size);
+  auto target = RenderTargetBuilder()
+                    .SetSize(size)
+                    .SetRenderTargetType(RenderTargetType::kOffscreen)
+                    .Build(*context.GetContext());
```

Comment:
This dropped the storage mode.

Reply:
(before this change the storage mode would end up as kDevicePrivate, now it ends up as kDeviceTransient)

Reply:
And a couple other defaults changed here too.

Reply:
sorry to make wrong default values, now changed back.